lib/remap-functions/src/get_env_var.rs

```diff
@@ -0,0 +1,92 @@
+use remap::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct GetEnvVar;
+
+impl Function for GetEnvVar {
+    fn identifier(&self) -> &'static str {
+        "get_env_var"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "name",
+            accepts: |v| matches!(v, Value::Bytes(_)),
+            required: true,
+        }]
+    }
+
+    fn compile(&self, mut arguments: ArgumentList) -> Result<Box<dyn Expression>> {
+        let name = arguments.required("name")?.boxed();
+
+        Ok(Box::new(GetEnvVarFn { name }))
+    }
+}
+
+#[derive(Debug, Clone)]
+struct GetEnvVarFn {
+    name: Box<dyn Expression>,
+}
+
+impl GetEnvVarFn {
+    #[cfg(test)]
+    fn new(name: Box<dyn Expression>) -> Self {
+        Self { name }
+    }
+}
```

Comment:
I'd just remove this, as it's only use in tests, and it's trivial to write `GetEnvVarFn { value }` instead.

I know we still have some functions that follow this pattern, but we've been moving away from it for a while now.